paymentsheet/src/main/java/com/stripe/android/link/LinkLaunchMode.kt

```diff
@@ -22,6 +22,11 @@ internal sealed interface LinkLaunchMode : Parcelable {
          * If true, shares the payment details immediately after creating it in passthrough mode.
          */
         val sharePaymentDetailsImmediatelyAfterCreation: Boolean = true,
+
+        /**
+         * Optional hint to be displayed.
+         */
+        val hint: String? = null,
```

Comment:
maybe we can add a bit more context? 

```suggestion

        /**
         * Optional hint to be displayed above the available payments list.
         */
        val hint: String? = null,
```

Reply:
I don't think so -- this `hint` should be agnostic to the design.